test/observer.spec.ts

```diff
@@ -319,7 +319,8 @@ describe('DataLayerObserver unit tests', () => {
   it('it should register and call an operator before the destination', () => {
```

Comment:
nit: maybe break this into two test cases: one for a single operator and one for multiple operators before destination?

Up to you.

Reply:
It’s a good idea; will update.